packages/contracts-bedrock/test/kontrol/proofs/utils/KontrolUtils.sol

```diff
@@ -4,78 +4,7 @@ pragma solidity 0.8.15;
 import { Vm } from "forge-std/Vm.sol";
 import { KontrolCheats } from "kontrol-cheatcodes/KontrolCheats.sol";
 
-// The GhostBytes contracts are a workaround to create a symbolic bytes array. This is slow, but
-// required until symbolic bytes are supported in Kontrol: https://github.com/runtimeverification/kontrol/issues/272
-contract GhostBytes {
-    bytes public ghostBytes;
-}
-
-contract GhostBytes10 {
-    bytes public ghostBytes0;
-    bytes public ghostBytes1;
-    bytes public ghostBytes2;
-    bytes public ghostBytes3;
-    bytes public ghostBytes4;
-    bytes public ghostBytes5;
-    bytes public ghostBytes6;
-    bytes public ghostBytes7;
-    bytes public ghostBytes8;
-    bytes public ghostBytes9;
-
-    function getGhostBytesArray() public view returns (bytes[] memory _arr) {
-        _arr = new bytes[](10);
-        _arr[0] = ghostBytes0;
-        _arr[1] = ghostBytes1;
-        _arr[2] = ghostBytes2;
-        _arr[3] = ghostBytes3;
-        _arr[4] = ghostBytes4;
-        _arr[5] = ghostBytes5;
-        _arr[6] = ghostBytes6;
-        _arr[7] = ghostBytes7;
-        _arr[8] = ghostBytes8;
-        _arr[9] = ghostBytes9;
-    }
-}
-
 /// @notice Tests inheriting this contract cannot be run with forge
 abstract contract KontrolUtils is KontrolCheats {
```

Comment:
Feels like this file can be deleted now to simplify things? Regular tests would then directly inherit from `KontrolCheats`, and they'll already have `vm`

Reply:
Oh I see, the `.k.sol` tests do not inherit from the regular foundry `Test` contract which includes `vm`. If it doesn't break anything, inheriting from that to get `vm` would be ideal, e.g. we'd have `contract L1ERC721BridgeKontrol is Test, DeploymentSummary, KontrolCheats`

Reply:
I'll test adding `Test` as you suggest! The rationale behind excluding it was to declutter the `.k.sol` files as much as possible, particularly not including the `Test` contract

Reply:
Bumping this comment